src/wallet/rpcwallet.cpp

```diff
@@ -1625,6 +1627,7 @@ UniValue listtransactions(const JSONRPCRequest& request)
             "2. count          (numeric, optional, default=10) The number of transactions to return\n"
             "3. skip           (numeric, optional, default=0) The number of transactions to skip\n"
             "4. include_watchonly (bool, optional, default=false) Include transactions to watch-only addresses (see 'importaddress')\n"
+            "5. omit_mnpayments   (bool, optional, default=false) Do not handle masternode outputs (improves performance)\n"
```

Comment:
Help string and variable name suggest we remove znode payments from the list which is not true, we just don't check if txout is znode payment.